bats/tests/helpers/vm.bash

```diff
@@ -66,18 +68,19 @@ EOF
 }
 
 start_container_runtime() {
+    local container_runtime="${1:-$RD_CONTAINER_RUNTIME}"
     rdctl start \
           --path-management-strategy rcfiles \
           --kubernetes.suppress-sudo \
           --updater=false \
-          --container-engine="$RD_CONTAINER_RUNTIME" \
+          --container-engine="$container_runtime" \
           --kubernetes-enabled=false \
           "$@" \
           3>&-
 }
 
 start_kubernetes() {
-    start_container_runtime \
-        --kubernetes-enabled \
-        --kubernetes-version "$RD_KUBERNETES_PREV_VERSION"
+    $RDCTL start \
+      --container-engine="$container_runtime" \
+      --kubernetes-enabled=false 3>&-
```

Comment:
Please don't make changes to bats! This will just create merge conflicts later. The current changes in this PR seem to be reverting recently merged change and look to be broken now.

I'll update BATS to use v5 in my next refactoring PR (assuming this PR gets merged first).

Reply:
ok, that happened due to a rebase conflict